modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/upgrade/CommerceProductContentWebUpgradeStepRegistrator.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.product.content.web.internal.upgrade;
+
+import com.liferay.commerce.product.content.web.internal.upgrade.v1_0_0.PortletPreferenceValueUpgradeProcess;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.upgrade.registry.UpgradeStepRegistrator;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Danny Situ
+ */
+@Component(
+	enabled = false, immediate = true, service = UpgradeStepRegistrator.class
+)
+public class CommerceProductContentWebUpgradeStepRegistrator
+	implements UpgradeStepRegistrator {
+
+	@Override
+	public void register(Registry registry) {
+		if (_log.isInfoEnabled()) {
+			_log.info(
+				"Commerce product content web upgrade step registrator " +
+					"started");
+		}
+
+		registry.register(
+			"0.0.0", "1.0.0", new PortletPreferenceValueUpgradeProcess());
```

Comment:
@andrea-ale-sbarra after looking into this I think leaving it as is should be ok.

Found this old grow article that gave me  a better understanding of how the version numbers work. https://grow.liferay.com/group/guest/people/-/wiki/34277/Upgrade+Process+-+Schema+Version+Definition.

> Auto Upgrade and initial upgrade process
>
> For non-service builder modules, the upgrade step from 0.0.0 to X.X.X is executed automatically even when the auto-upgrade property is set to false because it's considered as the initial upgrade where we can generate/modify some data needed for the module to work.
> 
> For service builder modules, the upgrade step from 0.0.0 to the latest schema version is automatically generated and reserved for the initial creation of the tables.

So I think since this upgrade is for a non service builder module and mainly run during a db upgrade it should always run from 7.3 -> 7.4. There shouldn't be any harm in running this in 7.4 either. Thanks!